### PR TITLE
Datalake in java and Datalake interface

### PR DESCRIPTION
**Pull request by JoelClemente**

### CODE REVIEW

**Comment by JoelClemente** [comment]
Ignore webscraperTest

**Review by BongoloZakolo — APPROVED** [review]

File: java/DataLake/files/pg72005.txt
Comment (adam-prochazka) [inline java/DataLake/files/pg72005.txt:1]: Do you think this file is necessary to be versioned?

File: java/DataLake/src/main/java/BookReader.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+     * @return The path separator (e.g., "/" or "\").
+     */
+    String getPathSeparator();
+
+    /**
+     * Read a book from a given path and return a list containing the book name and its words.
+     * @param path The path to the book file.
+     * @return A list containing the book name and its words.
+     */
+    List<Object> readBook(String path);
```
Comment (adam-prochazka) [inline java/DataLake/src/main/java/BookReader.java:22]: If you say you return a list containing the book name and its word, why do you return a `List` of **`Objects`**?

**Review by adam-prochazka — CHANGES_REQUESTED** [review]
Why do we have the `.iml` files?

File: java/DataLake/src/main/java/BookReader.java (hunk 1 above)
Comment (JoelClemente) [reply java/DataLake/src/main/java/BookReader.java:22]: Okey ill do this night

**Comment by carlillous** [comment]
It is fine to me. I approve de List<Object> return thing for reading the book, can't think in another "simple" way of doing this. Look for deleting the imls thing and the example book dowloaded.

**Comment by carlillous** [comment]
texted you what i think. try also looking if .iml files can be deleted and book file for test also could be deleted.

**Pull request closed without merging**